iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/dataset/opentrials/sqlite_builder/oozie_app/lib/scripts/buildopentrialsdb.sql

```diff
@@ -1,6 +1,9 @@
 drop table if exists opentrials;
 
-create table opentrials as select id,acronym from (select id,publisher,case when alternateidentifiers is null then "" else jsonpath(alternateidentifiers,publisher) end as acronym from (setschema 'id,publisher,alternateIdentifiers'select jdictsplit(c1,"id","publisher","alternateIdentifiers") from stdinput())) ;
```

Comment:
Hmm, were there any relevant changes meant to be introduced in this file? I tried to read word by word and it looks like the same command just separated with newlines. 

So I guess the vital and the only change was in `jsonpath.py` file, am I right?

Reply:
Yes, that is the actual change.